eZ/Publish/Core/IO/Tests/UrlDecorator/AbsolutePrefixTest.php

```diff
@@ -17,7 +18,12 @@ class AbsolutePrefixTest extends PrefixTest
 {
     protected function buildDecorator($prefix)
```

Comment:
nitpick: the change
``` diff
+ declare(strict_types=1);
```
implies that both here and for `provideData` method strict types are required (argument & return)
```suggestion
    protected function buildDecorator(string $prefix): UrlDecorator
```
(import for `UrlDecorator` required ^)